modules/network-observability-resources-table.adoc

```diff
@@ -0,0 +1,28 @@
+//module included in the following assemblies:
+// network_observability/configuring_operator.adoc
```

Comment:
```suggestion
// network_observability/configuring_operator.adoc
```

More of a nit than anything, but add a space after the commented out clauses with the doc's structural line entries. More for consistency with other modules.

Reply:
A really nitty nit, should be // Module

Reply:
Good ones, TY!